templates/index.html

```diff
@@ -66,7 +70,10 @@ <h1><a href="https://github.com/brentvollebregt/hit-counter">Hit Counter</a></h1
 
                     <h2 class="mt-4">Get the Image for the Page It's On</h2>
                     <p class="my-1">If you want to get the hit count image for a page you can edit and you want to count, add this line:</p>
-                    <p id="standard" class="text-primary my-0 o-a">&lt;img src="<script>host()</script>/count/tag.svg" alt="Hits"&gt;</p>
+                    <div class="bg-gray-700 mw-100 rounded py-2 px-4" style="display: inline-block">
+                        <pre class="hide-scrollbar"><code class="text-white">&lt;img src="<script>host()</script>/count/tag.svg" alt="Hits"&gt;</code></pre>
+                    </div>
```

Comment:
In terms of the scrollbar, if you do something like:

```
<div>
    <pre class="bg-gray-700 mw-100 rounded py-2 px-4">
        <code class="text-white">&lt;img src="<script>host()</script>/count/tag.svg" alt="Hits"&gt;</code>
    </pre>
</div>
```

the scrollbar will now be at the bottom of the coloured background and look a lot cleaner.